sdc-plugin/sdc.cc

```diff
@@ -192,18 +192,59 @@ struct GetClocksCmd : public Pass {
 	log("\n");
     }
 
+    std::vector<std::string> extract_list(const std::string& args) {
+	std::vector<std::string> port_list;
+	std::stringstream ss(args);
+	std::istream_iterator<std::string> begin(ss);
+	std::istream_iterator<std::string> end;
+	std::copy(begin, end, std::back_inserter(port_list));
+	return port_list;
+    }
+
     void execute(std::vector<std::string> args,
                  RTLIL::Design* design) override {
-	if (args.size() > 1) {
-	    log_warning("Command doesn't support arguments, so they will be ignored.\n");
+	bool generated_clocks(false);
```

Comment:
Can you please add some more comments through here?

Reply:
You mean the generated_clocks part. Added a comment we currently treat all added clocks as auto-generated.

Reply:
Am I missing something, or did you not add a comment?

Reply:
I don't know what type of comment should go there. The switch itself is described in the tool's [help](https://github.com/SymbiFlow/yosys-symbiflow-plugins/pull/52/files#diff-e71c67a80542c63bb4832fffed080e0d86d1cc0dee1ac1b0e255fd7f6520e0aeR192) and some comment about generated clocks [here](https://github.com/SymbiFlow/yosys-symbiflow-plugins/pull/52/files#diff-e71c67a80542c63bb4832fffed080e0d86d1cc0dee1ac1b0e255fd7f6520e0aeR213)

Reply:
Your code below this comment is doing a bunch of stuff.  What is it doing, and why it is doing it?

Reply:
Done